src/Uno.UI/UI/Xaml/Controls/MenuFlyout/MenuFlyout.cs

```diff
@@ -156,6 +156,8 @@ private protected override void OnOpening()
 				(menuFlyoutPresenter as IMenuPresenter).OwningMenu = parentMenu != null ? parentMenu : this;
 				menuFlyoutPresenter.UpdateTemplateSettings();
 
+				base.OnOpening();
```

Comment:
@MartinZikmund Do you know if the event should be fired if `menuFlyoutPresenter` is null?

Reply:
Indeed, for consistency it should. It seems that WinUI can't handle the `null` presenter anyway, but the event gets triggered before crashing.